plugins/generator-1.20.6/neoforge-1.20.6/mappings/enchantmenttypes.yaml

```diff
@@ -0,0 +1,58 @@
+_default: DIGGER

```

Comment:
I know this is closed, but for future reference, default value needs provide same number of mapping tables as other entries, 2 in this case